stories/theme/index.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  Title,
+  Description,
+  Heading,
+  Source,
+  Stories,
+} from '@storybook/addon-docs/blocks';
+import type { Meta } from '@storybook/react';
+import { BREAKPOINTS } from 'src/constants';
+
+export * from './boxShadows.stories';
+export * from './colors.stories';
+export * from './fonts.stories';
+export * from './typography.stories';
+
+const styledComponentSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  'export const StyledComponent = styled.div`\n  color: ${({ theme }) => theme.COLORS.primary};\n`';
+
+const cssPropSource =
+  // eslint-disable-next-line no-template-curly-in-string
+  '<Component css={(theme: ThemeType) => css`color: ${theme.COLORS.primary}`}>';
+
+export default {
+  title: 'Constants/Theme',
+  parameters: {
+    chromatic: { viewports: [BREAKPOINTS.xs] },
```

Comment:
```suggestion
    chromatic: { disable: true },
```

Since we'll effectively be evaluating the theme in every component story, I think we can skip snapshot testing on the actual theme stories themselves. 

I kept the story for the `Available Constants` in Chromatic since it groups everything on one big page, but since we're (wisely) breaking up the stories into smaller chunks for the theme, it'll push us closer to the limits of our Open Source snapshot allotment every month, so it's probably best to remove redundant visual tests.